solana/pyth2wormhole/client/src/main.rs

```diff
@@ -136,8 +136,10 @@ async fn main() -> Result<(), ErrBox> {
             
             let new_ops_owner = if remove_ops_owner {
                 None
+            } else if let Some(given_ops_owner) = ops_owner_addr {
```

Comment:
Nice catch! I think it would look its cleanest if we followed the `unwrap_or` pattern used by other Option<> values couple lines down